app/helpers/podcast_planner_helper.rb

```diff
@@ -46,24 +45,14 @@ def date_is_in_month?(date, month)
 
   def calendar_day_tag(day:, month:, calendar:, &block)
     data = {}
-    cls = []
+    cls = calendar.td_classes_for(day)
 
     if date_is_in_month?(day, month)
-      data[:controller] = DATE_CONTROLLER
+      data[:calendar_target] = CALENDAR_TARGET
       data[:action] = [TOGGLE_ACTION, RECOUNT_ACTION].join(" ")
-
-      is_new = date_is_in_dates?(day, @planner.dates)
-      is_existing = date_is_in_dates?(day, @draft_dates)
-      if is_new && is_existing
-        cls = ["bg-warning", "bg-danger", "text-light"]
```

Comment:
All colorizing of dates in the calendar had to move to the browser.  Since the days in those calendar represent a certain timezone, the "existing drafts" timestamps need to also be localized.